app/src/main/java/com/example/appmarvel/services/serviceimplementation/GetCharactersListServiceImpl.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.appmarvel.services.serviceimplementation
+
+import com.example.appmarvel.entity.Character
+import com.example.appmarvel.services.ServiceGenerator
+import com.example.appmarvel.services.api.MarvelApi
+import com.example.appmarvel.services.api.mapper.CharacterMapper
+import com.example.appmarvel.services.serviceinterface.GetCharactersListService
+import com.example.appmarvel.services.utils.Result
+
+class GetCharactersListServiceImpl : GetCharactersListService {
```

Comment:
the name should be something like "MarverServiceImpl" as you can use it for any endpoint on this api

Reply:
Done